tests/components/alarm_control_panel/test_manual.py

```diff
@@ -72,6 +72,11 @@ def test_arm_home_with_pending(self):
         self.assertEqual(STATE_ALARM_PENDING,
                          self.hass.states.get(entity_id).state)
 
+        self.assertTrue(self.hass.states.
```

Comment:
Don't break the line like this. Break it directly after one of the beginning parenthesis.